templates/repo/view_list.tmpl

```diff
@@ -34,7 +34,10 @@
 					</span>
 				{{end}}
 			</th>
-			<th class="text grey right age">{{if .LatestCommit}}{{if .LatestCommit.Committer}}{{TimeSince .LatestCommit.Committer.When $.i18n.Lang}}{{end}}{{end}}</th>
+			<th class="text grey right age">
+				<span class="mobile-align">
+					{{if .LatestCommit}}{{if .LatestCommit.Committer}}{{TimeSince .LatestCommit.Committer.When $.i18n.Lang}}{{end}}{{end}}</th>
+				</span>
```

Comment:
How can it be valid HTML if you close the `th` before the `span`?

Reply:
No idea 🤷🏽 nobody is bothered by it.

Reply:
Are you sure?
Is there a reason not to use valid HTML?
Because while major browsers might support it for some unknown reason, I don't think every implementation will.

Reply:
I was joking 😄 But it was indeed not caught by the browser or go, for some reason they weren't bothered by it.